apidoc/Facebook.yml

```diff
@@ -891,6 +891,46 @@ methods:
     since: "6.1.0"
     platforms: [iphone, ipad]
 
+  - name: logPushNotificationOpen
+    summary: Registers opened push with Facebook campaign. 
+    platforms: [iphone, ipad]
+    since: “6.1.0”
+    properties:
+      - name: value
+        summary: Dictionary of data received in push notifications callback
+        type: Dictionary
+
+  - name: setPushNotificationsDeviceToken
+    summary: dictionary of data received in the push notification
+    platforms: [iphone, ipad]
+    since: “6.1.0”
+    properties:
+      - name: value
+        summary: Device Token received when registering for push notifications
+        type: String
+
+  - name: logPushNotificationOpen
+    summary: Registers opened push with Facebook campaign. 
+    platforms: [android]
```

Comment:
Remove the other two iOS-methods and instead change this to `[iphone, ipad, android]`. Are tehy receiving the same parameters, so the docs can be used for both platforms?